iroh-base/src/key.rs

```diff
@@ -280,7 +280,7 @@ impl FromStr for PublicKey {
 #[derive(Clone)]
 pub struct SecretKey {
     secret: SigningKey,
-    secret_crypto_box: OnceCell<crypto_box::SecretKey>,
+    secret_crypto_box: OnceLock<crypto_box::SecretKey>,
```

Comment:
This wasn't stable back when we wrote this initially, right?

Reply:
I think so